src/consensus/malachite/network_connector.rs

```diff
@@ -224,14 +230,45 @@ where
                 //     return Ok(());
                 // }
 
-                trace!(%from, height = %status.height, "Received status");
+                trace!(%from, height = %status.tip_height, "Received status");
 
                 output_port.send(NetworkEvent::Status(
                     status.peer_id,
-                    Status::new(status.height, status.history_min_height),
+                    Status::new(status.tip_height, status.history_min_height),
                 ));
             }
 
+            Msg::NewEvent(Event::ConsensusMessage(channel, from, _)) => {
```

Comment:
What are these messages we're ignoring? Is it possible to enumerate the Channel::... explicitly so it's easier to understand?